cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/SqlToCqlTypeConverter.java

```diff
@@ -788,11 +797,54 @@ else if (object instanceof scala.Tuple2)
             }
             return map;
         }
+    }
+
+    public static class UdtConverter extends NullableConverter<Object>
```

Comment:
Why it converts to `Object` instead of `BridgeUdtValue`?

Reply:
Originally, I was trying to convert to the final values, but it ended up it wasn't possible due to some serialization issues, so it returns the `BridgeUdtValue` - I'll update it to be more specific.